src/resources/events.ts

```diff
@@ -142,6 +143,23 @@ class EventsResource extends ResourceClass {
 		}) as any;
 	}
 
+	/**
+	 * Delete an event
+	 * @auth true
+	 * @params {id:uuid}
+	 * @required {id:uuid}
+	 * @url /events/{id}
+	 */
+	delete(): EventInterface {
+		return createRequestMethod({
+			name: "delete",
```

Comment:
Keep the `del` alias around for backwards compatibility
```suggestion
			names: ["delete", "del"],
```